src/sysupdate/sysupdate-resource.c

```diff
@@ -724,6 +724,10 @@ int resource_resolve_path(
                 }
 
                 r = chase(rr->path, relative_to, chase_flags, &resolved, NULL);
+                if (r == -ENOENT) {
```

Comment:
Hmm, IIRC, `chase()` fails with `-ENOENT` only when the root directory (in this case `relative_to`) does not exist.
That sounds something a real bug and we should not ignore the failure, no?

Reply:
Ah, yeah, my bad. You are right. chase() returns -ENOENT when the file does not exist.
But returns 0 when CHASE_NONEXISTENT flag is set.

Reply:
No worries, and thanks for asking, it got us to a better fix!